sysutils/apcupsd/src/opnsense/mvc/app/controllers/OPNsense/Apcupsd/forms/general.xml

```diff
@@ -0,0 +1,139 @@
+<form>
+    <field>
+        <id>apcupsd.general.Enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help><![CDATA[Enable the APC UPS service.]]></help>
```

Comment:
You can remove CDATA when you dont use code like in general.Device

Reply:
OK, we could start cleanup CDATA entries then in strings not containing any code like <br> or similar.
https://en.wikipedia.org/wiki/CDATA
//Dan Lundqvist (mrzaz)